README.md

```diff
@@ -26,11 +26,13 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [AddCode](https://addcode-io.breezy.hr) [:rocket:](https://addcode-io.breezy.hr/#positions) | Software engineering. | `Porto` |
 | [Affinity](https://affinity.pt) [:rocket:](https://affinity.pt/en/careers/) | Consultancy, Outsourcing. | `Lisboa` `Porto` |
 | [AGAP2](http://www.agap2-it.pt/Agap2IT) [:rocket:](http://www.agap2-it.pt/Agap2IT/pt/pages/carreira/) | Consultancy, Outsourcing. | `Lisboa` `Porto` |
+| [Alter Solutions](https://www.alter-solutions.com/pt-pt/) [:rocket:](https://www.linkedin.com/company/alter-solutions-portugal/) | Consultancy, outsourcing and nearshore projects. | `Lisboa` `Porto` |
```

Comment:
The Linkedin link is not a careers link, can you please check?

Reply:
Right now Linkedin seems to be the best way for candidates to contact Alter Solutions. It has a carrers page but it is not actively managed by the Portuguese team I think. I can link it instead but LinkedIN has more value for the purpose. Let me know what you prefer.

Reply:
I have removed it because it is not a careers link. Thanks.